### PR TITLE
Update for site.php

### PR DESCRIPTION
**Pull request by tankerkiller125**
This update the install docs for the flat folder structure to the new `site.php` method.

### CODE REVIEW

**Comment by jordanjay29** [comment]
Moving `index.php` into the same directory as `site.php` will cause this line to become invalid and need correcting as well:

```
$site = require '../site.php';
```

**Comment by tankerkiller125** [comment]
I'll update when I get a chance tonight.

File: docs/install.md [hunk 1]
```diff
@@ -99,12 +99,9 @@ By default Flarum's directory structure includes a `public` directory which cont
...
 However, if you wish to host Flarum in a subdirectory (like `yoursite.com/forum`), or if your host doesn't give you control over your webroot (you're stuck with something like `public_html` or `htdocs`), you can set up Flarum without the `public` directory.
 
-Simply move all the files inside the `public` directory (including `.htaccess`) into the directory you want to serve Flarum from. Then edit `.htaccess` and uncomment lines 9-14 in order to protect sensitive resources. Finally, edit both `index.php` and the `flarum` executable, and update the paths in the following lines to reflect your new directory structure:
-
-```php
-require 'vendor/autoload.php';
-```
+Simply move all the files inside the `public` directory (including `.htaccess`) into the directory you want to serve Flarum from. Then edit `.htaccess` and uncomment lines 9-14 in order to protect sensitive resources. Finally, edit the `site.php` and update the paths in the following lines to reflect your new directory structure:
 
+site.php
```
Comment (franzliedke) [inline docs/install.md]: Is this intended?

Comment (tankerkiller125) [reply docs/install.md]: It was not, it's been removed in the latest commit.

**Pull request merged**